rfc-031-keep-support-docs-with-code

```diff
@@ -0,0 +1,33 @@
+# Keep support documentation with code
+
+## Summary
+
+Support and other technical documentation should be kept with the code it documents, so that it can be found easily.
+
+## Problem
+
+Support documentation exists in multiple places currently, and can be confusing. Some projects have `doc` directories, some don't, and some have multiple sources of truth. We should aim to standardise this.
+
+As well as making documentation easier to find, having it in standard locations makes future automation of support documentation easier.
+
+## Proposal
+
+All source code repositories MUST have a README.md file in the root. This file MUST include a link to support documentation under a `Support` heading.
+
+All repositories MAY contain a `doc` folder. 
+
+If it exists, the `doc` folder MUST include a file named `support.md` which acts as a first port of call for support users.
+
+If the repository has a `doc` folder, the README should link to the `support.md` file within it under this heading.
+
+If a repository does not contain a `doc` folder, the support section in the README MUST contain a link to support details, ideally to another related repository that includes `doc` folder.
```

Comment:
```suggestion
If a repository does not contain a `doc` folder, the support section in the README MUST contain a link to support details.
```

I'm not sure that adds much.

Reply:
Also, suggest "If a repository does not contain a documentation folder", in line with F's similar suggestions above.